OWNERS

```diff
@@ -0,0 +1,18 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+  - jistr
+  - archana203
+  - ciecierski
+  - frenzyfriday
+  - holser
+  - sathlan
+
+reviewers:
+  - archana203
+  - ciecierski
+  - fao89
+  - frenzyfriday
+  - holser
+  - jistr
+  - klgill
+  - sathlan
```

Comment:
We should also add @pinikomarov

Reply:
Pini added to reviewers. Should I add Pini to approvers too?

Reply:
I think we'll keep tweaking the lists e.g. based on who actually reviews patches, for now let's go with what's proposed here.